### PR TITLE
Fix(contact): Display LinkedIn profile as URL label in contact aside

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
## Problem

The full linkedin URL is displayed on contact aside instead of `LinkedIn profile`

## Solution

Upgrade RA to the latest version to support custom content on URL field

## How to test

Add a LinkedIn URL to a contact and check the contact aside

### CODE REVIEW

**Comment by arimet** [comment]
There is no development on linkdein, normal?

**Comment by jonathan-marmelab** [comment]
> There is no development on linkdein, normal?

This is normal, since the bug was due to the RA version that was not the latest (we did not add the problem on the demo since we always had the latest version)

**Review by arimet — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## How To Test
+
+_Describe the steps required to test the changes_
+
+## Additional Checks
+
+- [ ] The PR includes one or several **stories** (if not possible, describe why)
+- [ ] The **documentation** is up to date
+
+Also, please make sure to read the [contributing guidelines](https://github.com/marmelab/react-admin#contributing).
```
Comment (slax57) [inline .github/pull_request_template.md:18]: Good thing you added a PR template! \o/

But sections about the stories and contributing guidelines might not be applicable to this repo, so I'd remove them. Wdyt?

```suggestion
- [ ] The **documentation** is up to date
```

**Review by slax57 — COMMENTED** [review]
Otherwise the dev is fine :ok_hand:

**Review by slax57 — APPROVED** [review]

**Pull request merged**